ui/src/components/DashboardInventory.js

```diff
@@ -0,0 +1,150 @@
+//@flow
+
+import React from 'react';
+import { useQuery } from 'react-query';
+import styled, { css } from 'styled-components';
+import { Card, Loader } from '@scality/core-ui';
+import {
+  spacing,
+  fontSize,
+  fontWeight,
+} from '@scality/core-ui/dist/style/theme';
+
+import {
+  useAlertLibrary,
+  useHighestSeverityAlerts,
+  highestAlertToStatus,
+} from '../containers/AlertProvider';
+import { intl } from '../translations/IntlGlobalProvider';
```

Comment:
FYI: 
We've removed `IntlGlobalProvider`, but expose the `intl` throught the` <IntlProvider/>`. 
```suggestion
import { useIntl } from 'react-intl';
```